intermediate/xarray_and_dask.ipynb

```diff
@@ -1,14 +1,14 @@
 {
```

Comment:
<p>seems to be a raw cell by mistake</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>